tests/Microsoft.Bot.Builder.Prompts.Tests/TemperaturePromptTests.cs

```diff
@@ -39,10 +39,10 @@ public class TemperaturePromptTests
                             Assert.IsNotNull(tempResult.Text);
                             Assert.IsNotNull(tempResult.Unit);
                             Assert.IsInstanceOfType(tempResult.Value, typeof(float));
-                            context.Batch().Reply($"{tempResult.Value} {tempResult.Unit}");
+                            await context.SendActivity($"{tempResult.Value} {tempResult.Unit}");
```

Comment:
>SendActivity [](start = 42, length = 12)

This naming doesn't seem right
SendActivity() implies that we know who to send it to.  

I think the naming should be
```
Task<ResourceResponse> SendActivity(Activity)  
Task<ResourceResponse> Reply(text, ssml);
```
The point being SendActivity is fully routed activity (including proactive cases) and Reply() is specifically a reply to sender

Reply:
Per our discussion, this is a bigger picture issue we need to resolve across frameworks, and we're punting on this for today.